frontend/app_flowy/lib/workspace/presentation/home/menu/menu.dart

```diff
@@ -217,14 +217,22 @@ class MenuTopBar extends StatelessWidget {
             children: [
               renderIcon(context),
               const Spacer(),
-              FlowyIconButton(
-                width: 28,
-                onPressed: () => context
-                    .read<HomeBloc>()
-                    .add(const HomeEvent.collapseMenu()),
-                iconPadding: const EdgeInsets.fromLTRB(4, 4, 4, 4),
-                icon: svgWidget("home/hide_menu", color: theme.iconColor),
-              )
+              Tooltip(
+                  richMessage: TextSpan(children: [
+                    const TextSpan(text: "Close sidebar\n"),
```

Comment:
Should localize this string.  Check out this [link](https://appflowy.gitbook.io/docs/essential-documentation/contribute-to-appflowy/software-contributions/translation)